README.md

```diff
@@ -1,2 +1,9 @@
-# IVProyecto
-Asignatura IV UGR
+# Proyecto IV - Tienda de moda de ropa limitada
+
+Actualmente existe una gran especulación con el mercado de la ropa y zapatillas de marcas limitadas, lo que conlleva también un gran mercado de falsificaciones por detrás.
+
+
+Este proyecto tratará de facilitar la compra y venta de estos artículos utilizándose como intermediario y analizando la procedencia del producto.
+El comprado podrá visualizar los artículos por tipo de prenda y marca en la aplicación, también podrá visualizar el precio inicial del producto y su historial de ventas.
+El vendedor tendrá que ingresar datos del articulo que deseé vender, desde la talla hasta el ticket de compra o numero de serie.
+El negocio lo encontraríamos en la comisión de compra por parte del comprador y la comisión de verificar la originalidad del producto por parte del vendedor.
```

Comment:
Cuando se habla de lógica de negocio no se trata de decir de dónde se va a sacar el dinero (algo que estaría fuera del ámbito de la asignatura), aunque evidentemente la financiación del producto siempre da pistas sobre qué lógica de negocio se puede incluir. Se trata de incluir un procesamiento de datos no trivial que contribuya a resolver el problema del cliente. Eso es lo que tenéis que hacer para alcanzar este objetivo.